R/get_dataset_versions.R

```diff
@@ -0,0 +1,55 @@
+#' Get data set versions
+#'
+#' @inheritParams api_url
+#'
+#' @return Data frame listing all available versions of the given data set
```

Comment:
Could we do a more detailed breakdown of the columns and what they mean (for example, what is totalResults? Rows?)

Something like this format maybe?
```
#' @return A data frame with X columns:
#' \itemize{
#'   \item col1, something
#'   \item col2, something else
#' }
```

Reply:
Done